go/libkb/pipeowner_windows.go

```diff
@@ -68,13 +69,13 @@ func GetFileUserSid(name string) (*windows.SID, error) {
 	return userSID, nil
 }
 
-func IsPipeowner(name string) (bool, error) {
+func IsPipeowner(log logger.Logger, name string) (bool, error) {
```

Comment:
it would be nice to put debugs on the entry and exit of this function.

Reply:
I don't think that helps us but will do